### PR TITLE
fix: Full height on code editor [WEB-1258]

### PR DESCRIPTION
**Pull request by mapmeld**
## Description

Two fixes for code editor release:
- Make the container around the experiment code viewer fit the window height
- Removed `white-space: nowrap` CSS rule so file names wrap, instead of overlapping code editor

## Test Plan

Checkpoint file names with latest-main data, `/det/experiments/2744/code`
Screen Shot 2023-05-19 at 9 50 47 AM

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.tsx [hunk 1]
```diff
@@ -92,12 +92,14 @@ const ExperimentCodeViewer: React.FC<Props> = ({
   return (
     <React.Suspense fallback={<Spinner tip="Loading code viewer..." />}>
       <Spinner spinning={expFiles === NotLoaded} tip="Loading file tree...">
-        <CodeEditor
-          files={fileOpts}
-          readonly={true}
-          selectedFilePath={selectedFilePath}
-          onSelectFile={onSelectFile}
-        />
+        <div style={{ height: 'calc(100vh - 180px)' }}>
```
Comment (ClaireNeveu) [inline webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.tsx:95]: Please put this in a scss file so it's easier to fine later.

**Comment by keita-determined** [comment]
also, mobile view doesnt look the same as before.

image

**Comment by mapmeld** [comment]
++ to the margin
during the previous code editor PR, I removed the multiple UI options for mobile view, so currently we just have single-file or multi-file views for the code component. At this point I'm content that the code is browse-able

File: webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.codeContainer {
+  height: calc(100vh - 220px);
```
Comment (keita-determined) [inline webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss:2]: didnt notice it in the prev review, but when there's no one line log viewer, code viewer is out of window.

image

**Review by keita-determined — APPROVED** [review]
added one comment, but overall LGTM. hopefully, we can fix layout issue when new code viewer lands

File: webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss (hunk 2 above)
Comment (mapmeld) [reply webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss:2]: Multi-trial experiments have the additional row above tabs with searcher metric, searcher, etc.  So there could be some logic to shrink the code viewer to a different height on multi-trial?  I tried putting the height CSS on the containing Page (in ExperimentDetail) in an earlier version, but that affects scrolling in logs.

Comment (mapmeld) [reply webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss:2]: ok let me know  if the new css class works

**Pull request merged**